app/.env.development

```diff
@@ -5,3 +5,7 @@
 
 # If you deploy to a subpath, change this to the subpath so relative paths work correctly.
 NEXT_PUBLIC_BASE_PATH=
+
+# AWS Evidently Feature Flag variables
```

Comment:
@lorenyu kind of related to the identity provider conversation we had earlier: it could be nice if the template provides the AWS access and secret key so that someone could copy and paste into here to test this feature flag service locally, without setting up their own infra. Thoughts on this?

- Platform (manually?) creates an Evidently resource in AWS
- Platform (manually?) creates keys with access to that resources
- Platform has a 1password entry like `Next.js template local .env` with those secrets
- Somewhere(?) in the template docs it mentions how to set these env vars

Or am I over complicating this?

Reply:
FWIW for me the easiest way to test this branch was using the AWS credentials I had available, pasting them into terminal, and then running the app locally. So option 3 (1pass) sounds useful to me - are the credentials to login to the aws console associated w/ nava/platform work already available? If so, then what I did to test would be straightforward if we added some documentation for it. 

That said not my area of expertise and defer to others.

Reply:
there aren't really any AWS credentials we can use locally even if we used the infra template. the infra template grants permission to the ECS service, and the ECS service's credentials aren't something you can easily access or copy out, in fact i'm not even sure the ECS service has a long lived access key that is usable.

my thinking is that for MVP a developer shouldn't need to test against an actual instance of evidently. we should be able to provide a `LocalFeatureFlagManager` implementation that they can use to test out their application code, but they should be able to trust that the implementation using Evidently will work when they deploy it into infrastructure.

if a developer really wanted to test against an actual instance of Evidently from their local machine, then they should probably have an AWS IAM user anyways right? otherwise, what's the point of having access to Evidently if you can't go into AWS console and change around the flag settings and such. and if they already have console access, then they could grant themselves access to Evidently and then just use their personal AWS credentials.